flash_examples/predict/style_transfer.py

```diff
@@ -0,0 +1,47 @@
+import sys
+
+import torch
+
+from flash.utils.imports import _PYSTICHE_AVAILABLE
+
+if _PYSTICHE_AVAILABLE:
+    from pystiche import enc, loss, ops
+else:
+    print("Please, run `pip install pystiche`")
+    sys.exit(0)
+
+multi_layer_encoder = enc.vgg16_multi_layer_encoder()
+
+content_layer = "relu2_2"
+content_encoder = multi_layer_encoder.extract_encoder(content_layer)
+content_weight = 1e5
+content_loss = ops.FeatureReconstructionOperator(
+    content_encoder, score_weight=content_weight
+)
+
+
+class GramOperator(ops.GramOperator):
+    def enc_to_repr(self, enc: torch.Tensor) -> torch.Tensor:
+        repr = super().enc_to_repr(enc)
+        num_channels = repr.size()[1]
+        return repr / num_channels
```

Comment:
This change is only done to be compatible with the PyTorch example / paper. They both implement a double normalization by the number of channels which is most likely a bug. We could remove it here, but we would need to tune the hyperparameters then.

Reply:
from flash perspective I believe this is something that should happen internally in your library. /cc @tchaton